Header header
int32   frame_height
int32   frame_width
float32 inference_time
int32 	no_of_traffic_lights
int32[]  bounding_boxes
int32[]  class_ids
